apps/dotcms-ui-e2e/src/support/shared/Navigation.ts

```diff
@@ -0,0 +1,24 @@
+class Navigation {
+    static visit(url: string, options?: Cypress.VisitOptions) {
```

Comment:
Why not use `cy.visit` directly?

Reply:
I extracted this to the `Navigation` class so that in that way have all the basic methods into a single class (semantic)

but this can be discussed